integrationtests/endpoints-interop-it/src/test/java/org/infinispan/it/endpoints/BaseJsonTest.java

```diff
@@ -90,12 +87,18 @@ protected void setup() throws Exception {
       restEndpoint = String.format("http://localhost:%s/rest/%s", restServer.getPort(), CACHE_NAME);
    }
 
-   abstract String getEntityName();
+   protected String getMessageName() {
```

Comment:
```Message``` is protobuf speak for Entity. Let's continue to use Entity

Reply:
Updated.